rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/CompactView.java

```diff
@@ -553,54 +546,42 @@ private void init() {
             this.hoverLabel = new Label();
             this.hover.addMember(this.hoverLabel);
 
-            this.addDomHandler(new ContextMenuHandler() {
-                @Override
-                public void onContextMenu(ContextMenuEvent event) {
-                    event.preventDefault();
-                    event.stopPropagation();
-                }
+            this.addDomHandler(event -> {
+                event.preventDefault();
+                event.stopPropagation();
             }, ContextMenuEvent.getType());
 
-            this.addMouseOverHandler(new MouseOverHandler() {
-                @Override
-                public void onMouseOver(MouseOverEvent event) {
-                    if (dirty) {
-                        if (globalHover != null)
-                            globalHover.hide();
-                        dirty = false;
-                        if (node != null) {
-                            setHoverNodeLabel();
-                        } else if (host != null) {
-                            hoverLabel.setContents("<strong>Host</strong><br>" + host.getHostName());
-                        } else if (nodesource != null) {
-                            hoverLabel.setContents("<strong>NodeSource</strong><br>" + nodesource.getSourceName());
-                        }
-                        hover.moveTo(event.getClientX() - 155, event.getClientY() - 65);
-                        hover.show();
-                        globalHover = hover;
+            this.addMouseOverHandler(event -> {
+                if (dirty) {
+                    if (globalHover != null)
+                        globalHover.hide();
+                    dirty = false;
+                    if (node != null) {
+                        setHoverNodeLabel();
+                    } else if (host != null) {
+                        hoverLabel.setContents("<strong>Host</strong><br>" + host.getHostName());
+                    } else if (nodesource != null) {
+                        hoverLabel.setContents("<strong>NodeSource</strong><br>" + nodesource.getSourceName());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)